MinimalPluginStandard/Sniffs/DirectDBSniff.php

```diff
@@ -340,7 +341,7 @@ protected function find_assignments( $stackPtr, $var_name = null ) {
 			$var = $var_name;
 		}
 
-		return $this->assignments[ $context ][ $var ];
+		return $this->assignments[ $context ][ $var ] ?? false;
```

Comment:
This worked in my tests, but it'd be good for someone w/ more context to verify that it's the correct thing to do.

To test it, I added this to a plugin:

```php
<?php
global $wpdb;
$wpdb->get_results( "select foo where var = " . $_GET['foo'] );
```

...and then ran:

```sh
> ./bin/scan-dir.sh ~/vhosts/localhost/wp.test/public_html/content/plugins/basic-google-maps-placemarks/
FILE: /Users/iandunn/vhosts/localhost/wp.test/public_html/content/plugins/basic-google-maps-placemarks/views/front-end-head.php
----------------------------------------------------------------------------------------------------------------------------------------------------------
FOUND 1 ERROR AFFECTING 1 LINE
----------------------------------------------------------------------------------------------------------------------------------------------------------
 1 | ERROR | An error occurred during processing; checking has been aborted. The error message was: Undefined array key "$_GET['foo']" in
   |       | /Users/iandunn/vhosts/tools/wporg-code-analysis/MinimalPluginStandard/Sniffs/DirectDBSniff.php on line 345 (Internal.Exception)
```